config/config/sc-config.yaml

```diff
@@ -144,6 +144,8 @@ dex:
   tolerations: []
   affinity: {}
   nodeSelector: {}
+  insecureSkipEmailVerified: #bool
```

Comment:
Could you perhaps add a comment that this only affects the okta oidc connector?

Reply:
will do